e2e/playwright/authentication/exclude.tests.json

```diff
@@ -1,4 +1 @@
-{
-  "C213097" : "https://alfresco.atlassian.net/browse/ACS-5479",
```

Comment:
@akashrathod28 I'm approving this PR but can you ensure to close all jira tasks which were linked in these 3 exclude tests json files and have not more excluded tests as these tasks won't be needed more?

Reply:
Yes, as part of closing this ticket, I will close relavent assign jira.

Reply:
ok thank you